### PR TITLE
minor visual bug corrected

### PR DESCRIPTION
**Pull request by TheRealCL**
PLEASE VERIFY IF EVERYTHING ELSE IS   O.K.

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+  </div>
 
-
-  <h1>Pages#home</h1>
-  <p>Find me in app/views/pages/home.html.erb</p>
-
-
-
-
+  <br>
```
Comment (MBOZAR) [inline app/views/pages/home.html.erb:11]: Please review Br ?

File: app/assets/stylesheets/components/_containers.scss [hunk 2]
```diff
@@ -16,31 +16,32 @@
...
+text-decoration: none;
+box-shadow: 0 0 15px rgba(0,0,0,0.3);
+}
+
+.container-content-display a:hover {
+transform: scale(0.9, 0.9);
+box-shadow: 0 0 15px rgba(0,0,0,0.6);
+background: #6e8aa0;
+}
+
```
Comment (MBOZAR) [inline app/assets/stylesheets/components/_containers.scss:40]: Please review space

**Review by MBOZAR — CHANGES_REQUESTED** [review]
On en discute ce matin :)

**Review by MBOZAR — APPROVED** [review]

**Review by Rox1610 — APPROVED** [review]
ok

File: app/views/nvts/show.html.erb [hunk 3]
```diff
@@ -26,25 +26,4 @@
     <p><strong>Interested in collecting this item?</strong></p>
   <%= link_to 'Collect', transaction_path %>
   </div>
-
```
Comment (Rox1610) [inline app/views/nvts/show.html.erb:29]: contents delete from show page?

**Review by Rox1610 — CHANGES_REQUESTED** [review]
see comment on show page

**Pull request closed without merging**